### PR TITLE
uptodate: bump up versions

### PR DESCRIPTION
**Pull request by me3x**

### CODE REVIEW

File: micro-infra-spring-base/build.gradle [hunk 1]
```diff
@@ -31,9 +31,9 @@ dependencies {
     testCompile "org.spockframework:spock-core"
     testRuntime "org.spockframework:spock-spring"
     testCompile "org.springframework:spring-test"
-    testCompile 'org.hamcrest:hamcrest-library:1.3'
+    testCompile 'org.hamcrest:hamcrest-library:1.4-atlassian-1'
```
Comment (szpak) [inline micro-infra-spring-base/build.gradle]: Hmm, what is in that version?

Comment (me3x) [inline micro-infra-spring-base/build.gradle]: who knows :)

Comment (me3x) [inline micro-infra-spring-base/build.gradle]: It is not listed here:
https://code.google.com/p/hamcrest/downloads/list

However it looks that some libraries already use it:
https://www.versioneye.com/java/org.hamcrest:hamcrest-core/1.4-atlassian-1

Comment (szpak) [inline micro-infra-spring-base/build.gradle]: I think that "References: 1681" reflects to hamcrest-core in general, not that version. I would stick with 1.3 or check if version [2.0.0.0](https://github.com/hamcrest/JavaHamcrest/blob/master/CHANGES.txt) is (not) compatible with other libraries :).

**Comment by marcingrzejszczak** [comment]
The build still doesn't pass :/

**Comment by szpak** [comment]
It seems that wiremock started to return empty body instead of null.
```
com.ofg.infrastructure.web.resttemplate.custom.RestTemplateSpec > should handle exception thrown in getBody() during error handling when body empty FAILED

Condition not satisfied:

ex.body == null
| | |
| "" false

com.ofg.infrastructure.web.resttemplate.custom.ResponseException: Status code: [404], Headers: [[Transfer-Encoding:[chunked], Server:[Jetty(6.1.26)]]], Body: []
```

**Pull request merged**